pull_request_template.md

```diff
@@ -20,4 +20,14 @@
 
 <em>You may erase this after checking them all ;)</em>
 
-- [] CHANGELOG entry added
+- [ ] CHANGELOG entry added.
```

Comment:
Following on [Dan's comment](https://github.com/vtex-sites/gatsby.store/pull/12/files#r859083684), what do you think of specifying the changelog entry should go in the beginning of its section?

```suggestion
- [ ] Add an entry in `CHANGELOG.md` at the beginning of its due section. [The latest version should comes first](https://keepachangelog.com/en/1.0.0/#:~:text=The%20latest%20version%20comes%20first.).
```

Reply:
Just saw it's already there on line 24! My bad.

Reply:
No, worries @filipewl! I'm going to fix it.